### PR TITLE
Add TeamsMediaConnectivityPolicy cmdlets public docs

### PR DESCRIPTION
**Pull request by lirunping-MSFT**

### CODE REVIEW

**Comment by lirunping-MSFT** [comment]
@microsoft-github-policy-service agree company="{Microsoft}"

File: teams/teams-ps/teams/New-CsTeamsMediaConnectivityPolicy.md [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+schema: 2.0.0
+author: lirunping_MSFT
+ms.author: runli
+---
+
+
+# New-CsTeamsMediaConnectivityPolicy
+
+## SYNOPSIS
+This cmdlet creates an Teams media connectivity policy.
```
Comment (shivtorov) [inline teams/teams-ps/teams/New-CsTeamsMediaConnectivityPolicy.md:15]: "...creates **a** Teams ...policy..."

File: teams/teams-ps/teams/Get-CsTeamsMediaConnectivityPolicy.md [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+schema: 2.0.0
+author: lirunping_MSFT
+ms.author: runli
+---
+
+# Get-CsTeamsMediaConnectivityPolicy
+
+## SYNOPSIS
+
+This cmdlet retrieves all Teams media connectivity policy for current tenant.
```
Comment (shivtorov) [inline teams/teams-ps/teams/Get-CsTeamsMediaConnectivityPolicy.md:15]:
```suggestion
This cmdlet retrieves all Teams media connectivity policies for current tenant.
```

File: teams/teams-ps/teams/Grant-CsTeamsMediaConnectivityPolicy.md [hunk 3]
```diff
@@ -0,0 +1,189 @@
...
+### Example 3
+```powershell
+PS C:\> Grant-CsTeamsMediaConnectivityPolicy -Global -PolicyName Test
+```
+
+Assigns a given policy to the tenant.
+
+### Example 3
+```powershell
+PS C:\> Grant-CsTeamsMediaConnectivityPolicy -Global -PolicyName Test
```
Comment (shivtorov) [inline teams/teams-ps/teams/Grant-CsTeamsMediaConnectivityPolicy.md:71]:
```suggestion
PS C:\> Grant-CsTeamsMediaConnectivityPolicy -Global -PolicyName $null
```

Comment (lirunping-MSFT) [reply teams/teams-ps/teams/Grant-CsTeamsMediaConnectivityPolicy.md:71]: The "Test" in the example is an identify for the policy name. May I ask why it has to be $null ? Does it mean I need to change all the "Test" to $null in the examples of this file ?

Comment (lirunping-MSFT) [reply teams/teams-ps/teams/Grant-CsTeamsMediaConnectivityPolicy.md:71]: I'll keep this as-is. @shivtorov please let me know if you prefer this to be changed throughout the file.

File: teams/teams-ps/teams/Grant-CsTeamsMediaConnectivityPolicy.md [hunk 4]
```diff
@@ -0,0 +1,189 @@
...
+
+
+### Example 3
+```powershell
+PS C:\> Grant-CsTeamsMediaConnectivityPolicy -Global -PolicyName Test
+```
+
+Assigns a given policy to the tenant.
+
+### Example 3
```
Comment (shivtorov) [inline teams/teams-ps/teams/Grant-CsTeamsMediaConnectivityPolicy.md:69]:
```suggestion
### Example 4
```

File: teams/teams-ps/teams/New-CsTeamsMediaConnectivityPolicy.md [hunk 5]
```diff
@@ -0,0 +1,68 @@
...
+### Example 1
+```powershell
+PS C:\> New-CsTeamsMediaConnectivityPolicy -Identity Test
+
+Identity DirectConnection
+-------------------------
+Tag:Test Enabled
+```
+
+Returns the current value of "DirectConnection" policy with identity of "Test".
```
Comment (shivtorov) [inline teams/teams-ps/teams/New-CsTeamsMediaConnectivityPolicy.md:37]: It looks like the cmdlet "creates" instead of "returns"...

Comment (lirunping-MSFT) [reply teams/teams-ps/teams/New-CsTeamsMediaConnectivityPolicy.md:37]: Yes. It firstly create then returns(actually prints) the newly created policy. I'll update to make it more clear.

**Comment by lirunping-MSFT** [comment]
@shivtorov Kindly request for further review. Thanks!
